sql/core/src/test/scala/org/apache/spark/sql/execution/QueryPlanningTrackerEndToEndSuite.scala

```diff
@@ -38,4 +40,22 @@ class QueryPlanningTrackerEndToEndSuite extends SharedSparkSession {
     assert(tracker.rules.nonEmpty)
   }
 
+  test("SPARK-29227: Track rule info in optimization phase in streaming") {
```

Comment:
For the new feature, we don't use Apache JIRA issue prefix. Please remove `SPARK-29227: `.

Reply:
@dongjoon-hyun Thanks for your reply.
I fell confused about test name if just delele `SPARK-29227: `, cause this ut is to test tracker phase and rule info in streaming, not only rule info in optimization phase as jira title described. So should I delete `SPARK-29227: Track rule info in optimization phase in `?
And also, do you mean add Apache JIRA issue info(prefix and title) in test name if it is bugfix?

Reply:
Let me just merge it since it's rather a nit.

Reply:
No, @wenxuanguan ~ Only the JIRA ID.
> And also, do you mean add Apache JIRA issue info(prefix and title) in test name if it is bugfix?

Reply:
> No, @wenxuanguan ~ Only the JIRA ID.
> 
> > And also, do you mean add Apache JIRA issue info(prefix and title) in test name if it is bugfix?

@dongjoon-hyun got it, Thanks a lot.